### DISCUSSION TITLE
Intermediate HTML & CSS: New project

### DISCUSSION THREAD

**Discussion by nik-rev** [TheOdinProject/curriculum#27150] [Ideas]
I just completed this course and I feel like it would benefit from a project where we use tables to make a website layout. 

The most practice given in the tables tables section was an exercise from MDN about planet data, but I honestly feel that this isn't enough to really cement all the new information and should be just a warmup for a more interesting project.

**Comment by nik-rev** [comment]
Closing this because I don't think this is a good idea anymore

**Reply by NathanielAddo** [reply]
I am actually interested in the idea.

**Reply by nik-rev** [reply]
> I am actually interested in the idea.

The assignment from MDN for the planets data is already enough practice in my opinion.

But if you have some reasoning, I would love to discuss this with you!

**Reply by NathanielAddo** [reply]
Alright
